upi/openstack/03_bootstrap.yaml

```diff
@@ -0,0 +1,35 @@
+# Required Python packages:
+#
+# ansible
+# openstacksdk
+# netaddr
+
+- hosts: all
+
+  vars:
+    bootstrap_data: "{{ os_infra_id }}-bootstrap-ignition.json"
+
+    bootstrap_server: "{{ os_infra_id }}-bootstrap"
+    bootstrap_port: "{{ os_infra_id }}-bootstrap-port"
+
+  tasks:
+  - name: 'Create the bootstrap server port'
+    os_port:
+      name: "{{ bootstrap_port }}"
+      network: "{{ os_network }}"
+      security_groups:
+      - "{{ os_sg_master }}"
+      allowed_address_pairs:
+      - ip_address: "{{ os_subnet_range | next_nth_usable(5) }}"
+      - ip_address: "{{ os_subnet_range | next_nth_usable(6) }}"
+      - ip_address: "{{ os_subnet_range | next_nth_usable(7) }}"
+
+  - name: 'Create the bootstrap server'
+    os_server:
+      name: "{{ bootstrap_server }}"
+      image: "{{ os_image_rhcos }}"
+      flavor: "{{ os_flavor_master }}"
+      userdata: "{{ lookup('file', bootstrap_data) }}"
+      auto_ip: no
```

Comment:
I think we actually want `auto_ip: yes`. Both IPI and the UPI doc do assign a floating IP to the bootstrap node. It is used for `must-gather` in the case of a bootstrap failure.

I'd prefer if you kept it in for now (to keep parity with the UPI text as it stands).